server/src/test/java/org/elasticsearch/gateway/MetaDataStateFormatTests.java

```diff
@@ -416,6 +415,7 @@ public void testFailRandomlyAndReadAnyState() throws IOException {
     }
 
     private static MetaDataStateFormat<MetaData> metaDataFormat() {
+        //TODO re-write tests not to use meta data format
```

Comment:
I don't think we need this as a TODO.

Reply:
++

Reply:
https://github.com/elastic/elasticsearch/pull/35199/commits/d4250c32e9658ff48a226d53532b21ce4ad5f4ee